### DISCUSSION TITLE
Package no longer maintained

### DISCUSSION THREAD

**Discussion by mattiaskagstrom** [dotMorten/Microsoft.SqlServer.Types#66] [General]
It would seem that this package is no longer being maintained and have several dependencies to package versions with known vulnerabilities. (sqlclient 4.5.1)

**Comment by dotMorten** [comment]
Just because there isn’t new commits doesn’t imply that. It would be better to log an issue about the vulnerabilities so the maintainer can address it, instead of logging a passive aggressive issue like this.

**Reply by mattiaskagstrom** [reply]
https://github.com/dotMorten/Microsoft.SqlServer.Types/pull/58

Since you stopped responding for over a year, i just assumed you were no longer active. sorry about that.

**Reply by mattiaskagstrom** [reply]
I have opened an issue for my specific issue. I was unable to clone and fix it myself since I'm in a corporate network that blocks traffic from GitHub :(

https://github.com/dotMorten/Microsoft.SqlServer.Types/issues/67

**Reply by dotMorten** [reply]
The library does what I need and works, so haven’t needed to improve it. Security vulnerabilities is a different matter. having said that you can just explicitly reference a newer dependency and you should be good to go
